opennms-doc/guide-admin/src/asciidoc/text/bsm/bsm-admin.adoc

```diff
@@ -0,0 +1,125 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+_OpenNMS_ provides a very powerful and fine grained service assurance layer based on the _Pollerd_ daemon.
+The service assurance layer monitors and tracks the status of nodes, interfaces and services.
+The status of a managed entity is determined based on related events, alarms and outages.
+However, this approach treats every element of the system individually and there is no concept of relationships between different nodes or services in the service assurance layer.
+
+The _Business Service Monitoring_ (BSM) feature adds the ability to model relationships between managed entities.
+It combines sets of managed entities into hierarchically structured _Business Services_ that represent a high level view on its elements.
+The _Business Service'_ operational status is calculated from the alarm states of its associated elements determined by the service assurance layer.
+Of course, the operational status of a _Business Service_ can be checked at any time.
+If the operational status of a _Business Service_ is changing corresponding events are fired.
+
+==== Structure of a Business Service
+Business Services can contain attributes, _Reduction Keys_, references to monitored services and/or other business services.
+
+===== Reduction Keys as part of a Business Service
+_Business Services_ enables users to correlate the alarm based status of managed entities into its operational status.
+_Business Services_ can contain a list of _Reduction Keys_ that are used to identify the relevant alarms to summarize.
+_Reduction Keys_ can be added to business services as plain text.
+
+====== Monitored services as part of a Business Service
+_Business Services_ can contain references to monitored services (node/interface/service combination).
+A referenced monitored service is called _IP Service_ in the _Business Service_ architecture.
+_IP Services_ in a business service are interpreted as two _Reduction Keys_:
+
+* `uei.opennms.org/nodes/nodeLostService::%nodeID%:%ipAddress%:%serviceName%`
+-- This _Reduction Key_ matches the alarms fired if the monitored service goes down.
+* `uei.opennms.org/nodes/nodeDown::%nodeID%`
+-- This _Reduction Key_ matches the alarms fired if the entire node providing the monitored-service goes down.
+
+====== Business Services as part of Business Services
+_Business services_ can form relationships woth other _Business Services_.
+This offers the ability to create hierarchically structures of _Business Services_.
+To determine the operational status of a enclosing _Business Service_ the operational status of all its enclosed _Business Services_ is summarized.
+
+====== Attributes as part of Business Services
```

Comment:
please add new lines after heading to be consistent

Section with just one sentence?